src/p6/CoreBundle/Resources/views/Index/index.html.twig

```diff
@@ -3,31 +3,47 @@
 {% block title %}{{ parent() }}{% endblock %}
 
 {% block body %}
+<section class="header">
+    <div class="snowframe container">
+        <div class="snowfond">
+            <div class="snowname indie hit-the-floor">
+                <p class="boxhook">La référence par les pratiquants</p>
+            </div>
+        </div>
+    </div>
+    <div class="arrowtricks">
+        <a href="#tricks"><i class="logo-arrow fas fa-arrow-down"></i></a>
+    </div>
+</section>
 <section id="tricks">
 
 <div class="container mobile">
 {% for trick in tricklist %}
     <div class="row trickblock">
         <div class="col-xs-offset-1 col-xs-10 minitest"></div>
-        <div class="col-xs-offset-1 col-xs-6 merribold">{{ trick.name }}</div>
+        <div class="col-xs-offset-1 col-xs-6 merribold"><a href="{{ path('trick', {"id": trick.id }) }}">{{ trick.name }}</a></div>
+        {% if is_granted('ROLE_USER') %}
         <div class="col-xs-2"><i class="fas fa-pen blue"></i></div>
         <div class="col-xs-2"><i class="far fa-trash-alt red"></i></div>
+        {% endif %}
     </div>
     {% endfor %}
 </div>
 
 <div class="container desktop">
     <div class="row">
-{% set nbtrick = 5 %}
+{% set nbtrick = 15 %}
```

Comment:
So, what if we have more than 15 tricks to display ?

Reply:
line doesnt exist anymore. Replace by function.